lib-database/src/commonMain/kotlin/com/ustadmobile/core/db/UmAppDatabase.kt

```diff
@@ -38,11 +38,17 @@ import kotlin.jvm.Volatile
     //TODO: DO NOT REMOVE THIS COMMENT!
     //#DOORDB_TRACKER_ENTITIES
 
-], version = 36)
+], version = 37)
```

Comment:
Let's not increase the database version here. There are other fields that need migrated. I think we are going to have to a diff of lib-database-entities before we merge dev-crud. That would show us what fields have been added/removed.